src/services/helpers/utils/customErrorMessages.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+	ACTIVATION_LINK_EXPIRED: 'ACTIVATION_LINK_EXPIRED',
+	ACTIVATION_LINK_INVALID: 'ACTIVATION_LINK_INVALID',
```

Comment:
since these are very specific to the activation link service, these should also be defined in the activation link service.
Imagine we would pull out the activation service from the server and make it a microservice, we woulndt want to search these here.

Reply:
good point 👍